sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcRelation.scala

```diff
@@ -162,6 +162,10 @@ private[sql] class OrcRelation(
   extends HadoopFsRelation(maybePartitionSpec, parameters)
   with Logging {
 
+  /* org.apache.orc contains OrcConf which can be used later */
+  private[sql] val ORC_BLOOM_FILTER_COLUMNS = "orc.bloom.filter.columns"
```

Comment:
it is really confusing to have ORC_BLOOM_FILTER_COLUMNS and orcBloomFilterCols in the same class.

I'd just inline the string in orcBloomFilterCols. Same for orcBloomFilterFpp.